cli/command/container/opts.go

```diff
@@ -557,6 +562,7 @@ func parse(flags *pflag.FlagSet, copts *containerOptions, serverOS string) (*con
 		Ulimits:              copts.ulimits.GetList(),
 		DeviceCgroupRules:    copts.deviceCgroupRules.GetAll(),
 		Devices:              deviceMappings,
+		DeviceRequests:       deviceRequests,
```

Comment:
nit: looks like it could even be inlined here

```suggestion
		DeviceRequests:       copts.gpus.Value(),
```